src/objects/Offer.ts

```diff
@@ -55,6 +55,12 @@ export class Offer implements IOffer {
     return tradeItems;
   }
 
+  //TODO: Implement this method
+  // https://github.com/InternetEnemies/combatcritters-ts/issues/61
+  public async accept(): Promise<boolean> {
+    return Math.random() < .5;
+  }
```

Comment:
why would this be boolean, it should probably throw something if the purchase fails

Reply:
You wouldn't want to throw something if the purchase failed because you didn't have enough cards or currency

Reply:
I feel like the front end should probably check if the user can purchase it and gray out the buy button if they cant

Reply:
Yea you're right. I think that logic should maybe be in the wrapper then. Have a function in offer that returns whether or not the user has the items for the offer

Reply:
we might want it to be added to the compare function, have it return an object that has the missing items and also a `canPurchase` bool

Reply:
So have it return a 
```typescript
interface {
   tradeITems[]: ITradeItem<ICard | IPack | ICurrency>; 
   canPurchase: boolean;
 }
```

Reply:
is `tradeItems` meant to be the missing items, then yeah

Reply:
yea I might try to think of a better name for that lol